doc/testing.md

```diff
@@ -135,10 +135,19 @@ the first N tiers they can afford to run, but at least tier1.
 
 A brief description of the tiered test groups:
 
-- `tier1`: This is the lowest test tier. Multiple developers run these tests
+- `tier1`: This is the most fundamental test tier.
+  Roughly speaking, a failure of a test in this tier has the potential to
+  indicate a problem that would affect many Java programs. Tests in `tier1` include
+  tests of HotSpot, core libraries in the `java.base` module, and the `javac` compiler.
+  Multiple developers run these tests
   every day. Because of the widespread use, the tests in `tier1` are
   carefully selected and optimized to run fast, and to run in the most stable
-  manner. The test failures in `tier1` are usually followed up on quickly,
+  manner.
+  As a guideline, nearly all individual tests in `tier1` should complete in less than ten seconds
+  when run on common configurations used for development. Long-running tests,
```

Comment:
"should be complete" is okay, an alternative might be "are expected to complete" so better get across that it is aspirational.

Reply:
Updated wording.

Reply:
Thanks for the explanation, for providing numbers, and for updating the test.